packages/lazy-images/package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "@automattic/jetpack-lazy-images",
+	"version": "1.0.0",
+	"description": "Lazy Image Loading.",
+	"main": "dist/lazy-images.js",
+	"sideEffects": true,
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/jetpack.git",
+		"directory": "packages/lazy-images"
+	},
+	"private": true,
+	"author": "Automattic Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/issues"
+	},
+	"homepage": "https://github.com/Automattic/jetpack",
+	"scripts": {
+		"clean": "npx rimraf dist",
```

Comment:
What does this do that `"rm -rf dist"` doesn't?

Other than download and run code from the Internet, of course.

Reply:
It's cross-platform, so it should also work on Windows -- at least that's why we're using it in Calypso, which is where I copied it from :slightly_smiling_face:

Reply:
I'm not sure how much of Jetpack's stuff is intended to be run on Windows, versus gets run inside a Linux docker container anyway. I do see that the "clean" scripts in Jetpack's package.json all just use `rm -rf`.

If you are going to use a node thing, IMO it would be better to just declare it as a dependency instead of having npx download an arbitrary version at runtime.

Reply:
Happy to use `rm -rf`, if that's what JP does for everything else :+1: